lldb/source/Plugins/SymbolFile/DWARF/DWARFIndex.h

```diff
@@ -115,6 +122,11 @@ class DWARFIndex {
   bool
   GetFullyQualifiedTypeImpl(const DWARFDeclContext &context, DWARFDIE die,
                             llvm::function_ref<bool(DWARFDIE die)> callback);
+
+  /// Check if the type \a die can meet the requirements of \a query.
+  bool
+  ProcessTypeDieMatchQuery(TypeQuery &query, DWARFDIE die,
```

Comment:
Looks like other function names use all cap for DIE
```suggestion
  ProcessTypeDIEMatchQuery(TypeQuery &query, DWARFDIE die,
```